airbyte-ci/connectors/pipelines/pipelines/cdk/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
```

Comment:
The way `aircmd` works is that each plugin is its own poetry package, so doing so would require the entire airbyte-ci package to be a plugin that gets loaded, which might be chicken-and-egg. since airbyte-ci is the main cli entrypoint. In the "eventual" land where we share the same entrypoint then they could easily be all one package. But it might be possible to do with the current impl. I will think about this a bit.

The reasoning behind this design goes back to your and my original "developer vs operator" discussion and the abstraction that `aircmd` tries to solve with it. The idea is that people will only install plugins that are relevant to them, so as to not pollute their own global command space. I guess in your case the intersection of people who run airbyte-ci connectors and airbyte-ci cdk is pretty dang high so maybe it does make sense to say that they are one in the same. So if it's technically difficult to do now due to airbyte-ci being its own entrypoint (which it might not be, I'll check), we can at least say that yes, connectors and cdk are conceptually part of the same 'plugin'